tests/integration/utils.go

```diff
@@ -263,7 +263,7 @@ func executeTestCase(
 		logAttrs = append(logAttrs, corelog.Any("KMS", kms))
 	}
 
-	log.InfoContext(ctx, testCase.Description, logAttrs...)
+	log.InfoContext(ctx, "", logAttrs...)
```

Comment:
question: Are people happy with this? 

I can even remove this completely or log something else.